Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.11
+WORKDIR /app
+
+COPY poetry.lock pyproject.toml /app/
```

You also need to copy the app dir here:
```suggestion
COPY poetry.lock pyproject.toml /app/
COPY app /app/app
```

---

this one's related to the comment above

---

Yes, but this is acutally needed. WIthout it the build docker image is useless for production.

---

If this is conflicting with using this image for development, you should create a separate Dockerfile for development.
The main Dockerfile should be usable for deployment.

---

ah okay, should it be `COPY . /app`?

---

No, since we don't need stuff like the `tests` directory in the docker image. 
You would either have to create a dockerignore file or only copy the relevant parts into the docker image.